tests/tracking/request_header/test_registry.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+from unittest import mock
+from importlib import reload
+
+import mlflow.tracking.request_header.registry
+from mlflow.tracking.request_header.registry import (
+    RequestHeaderProviderRegistry,
+    resolve_request_headers,
+)
+from mlflow.tracking.request_header.databricks_request_header_provider import (
+    DatabricksRequestHeaderProvider,
+)
+
+# pylint: disable=unused-argument
```

Comment:
Are we supplying this due to `tmp_wkdir` below?

Reply:
that and the mock provider fixture